docs/spec/historical.rst

```diff
@@ -0,0 +1,291 @@
+.. _historical:
```

Comment:
Maybe add `from __future__ import annotations` to this page? I was reminded of it in the section about forward references. I think it's within the scope of the typing spec, since it changes semantics: even if you don't use runtime introspection, enabling it changes what gets accepted.

Reply:
I'm not sure it does; shouldn't type checkers infer the same types with and without the future import? Obviously the future import affects the runtime behavior, but this spec doesn't need to describe that.

Reply:
Huh, I suppose. Does the spec not describe *any* runtime behavior?

Reply:
It does in a few cases, but in principle I think it shouldn't. docs.python.org describes how Python the language works; this spec should only describe the semantics of the type system.